Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -1335,6 +1388,14 @@ Optional<CodePointDecomposition const> code_point_decomposition_by_index(size_t
     auto const& mapping = s_decomposition_mappings[index];
     return CodePointDecomposition { mapping.code_point, mapping.tag, ReadonlySpan<u32> { s_decomposition_mappings_data.data() + mapping.decomposition_index, mapping.decomposition_count } };
 }
+
+BidirectionalClass bidirectional_class(u32 code_point)
+{
+    if (auto const* entry = binary_search(s_bidirectional_classes, code_point, nullptr, CodePointBidiClassComparator {}))
+        return entry->bidi_class;
+
+    return {};
```

Comment:
Not sure about returning a default-initialized value here. That just means we return whatever happens to be the first bidi class alphabetically. Does the spec indicate we should have any specific default? Or maybe we should return an empty `Optional`?

Reply:
I can't find anything about a default, so I've gone with returning an empty Optional.